sdk-core/custom_tokens.md

```diff
@@ -14,17 +14,49 @@ class CustomTokenComputer:
     // The constructor is not captured by the specs; it's up to the implementing library to define it.
 
     // Returns token.nonce == 0.
-    is_fungible_token(token: Token): boolean;
+    // Can throw:
+    // - ErrInvalidCustomTokenIdentifier
```

Comment:
I don't think this should throw `ErrInvalidCustomTokenIdentifier` since you'll only check the nonce property of the `CustomToken` dto.

Reply:
Indeed, fixed :+1: